File: beyhive-backend/models/Outfit.js
const mongoose = require('mongoose');

const OutfitSchema = new mongoose.Schema({
  id: { type: String, required: true, unique: true },
  name: { type: String, required: true },
  location: { type: String, required: true },
  imageUrl: { type: String, default: null },
  isNew: { type: Boolean, default: false },
  section: { type: String, required: true },
  description: { type: String, default: null }
});

module.exports = mongoose.model('Outfit', OutfitSchema); 

File: android-app/app/src/main/java/com/beyhivealert/android/data/Outfit.kt
package com.beyhivealert.android.data

import kotlinx.serialization.Serializable

@Serializable
data class Outfit(
    val id: String,
    val name: String,
    val location: String,
    val imageName: String? = null, // Asset name (for backward compatibility)
    val imageUrl: String? = null, // Remote URL (new format)
    val isNew: Boolean,
    val section: String, // e.g. "Houston", "Washington", "Los Angeles", "Other"
    val description: String? = null
)

@Serializable
data class OutfitsResponse(
    val outfits: List<Outfit>
)
